examples/electricity-trade/cleanup.sh

```diff
@@ -0,0 +1,28 @@
+docker stop sawtooth-shell-default
+docker stop sawtooth-settings-tp-default
+docker stop sawtooth-xo-tp-python-default
+docker stop sawtooth-devmode-engine-rust-default
+docker stop sawtooth-rest-api-default
+docker stop sawtooth-intkey-tp-python-default
+docker stop sawtooth-validator-default
+docker stop geth1
+docker stop gethtestnet_init-chain-geth1_run_1
+docker stop gethtestnet_init-chain-geth1_run_2
+
+docker rm sawtooth-shell-default
+docker rm sawtooth-settings-tp-default
+docker rm sawtooth-xo-tp-python-default
+docker rm sawtooth-devmode-engine-rust-default
+docker rm sawtooth-rest-api-default
+docker rm sawtooth-intkey-tp-python-default
+docker rm sawtooth-validator-default
+docker rm geth1
+docker rm gethtestnet_init-chain-geth1_run_1
```

Comment:
Would you remove "docker rm gethtestnet_init-chain-geth1_run_1"? The reason is like the above.